pkg/combustion/templates/19-copy-os-files.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+cp -R ./os-files/* /
```

Comment:
Admittedly, it feels a bit weird to shell out for a copy command. My rationale is that:

* I couldn't find a great library that did it, and implementing it in Go is tedious.
  * Even if I found a library, it's one more dependency to run through the legal and auditing process
* We already shell out for a number of things, so it felt less dirty. If this was the first instance of shelling out in EIB, I would have tried harder to avoid it.

Reply:
Selinux context needs to be maintained I guess... I'm AFK but I think there is a flag on cp for that purpose.
Also what about rsync instead? 😅 (Maybe it just increases the footprint and make things more complex)

Reply:
But, don't we already have a [CopyFiles](https://github.com/suse-edge/edge-image-builder/blob/main/pkg/fileio/file_io.go#L80) function in EIB that can do exactly this?